opentargets_pharmgkb/evidence_generation.py

```diff
@@ -183,6 +183,17 @@ def get_functional_consequences(df):
         for variant_id, gene_id, gene_symbol, consequence_term in batch
         for genotype_id in vep_id_to_genotype_ids[variant_id]
     ]).drop_duplicates()
+    # For every VEP id, also record the no_sequence_alteration consequence for the corresponding ref/ref genotype
+    ref_ref_consequences = pd.DataFrame(data=[
+        {
+            'genotype_id': vep_id_to_ref_ref_id(variant_id),
+            'overlapping_gene': gene_id,
+            'consequence_term': 'no_sequence_alteration'
+        }
+        for batch in all_consequences
+        for variant_id, gene_id, _, _ in batch
+    ]).drop_duplicates()
```

Comment:
This will add a `ref/ref` genotype id for every variant but we only want to have it for the clinical annotations that contains a `ref/ref` annotation text. 
If I understand correctly, some of these will get filtered out on the `pd.merge` command bellow. 
Is that right ?

Reply:
Yes, that's correct